ingest/cell_metadata.py

```diff
@@ -35,7 +36,7 @@ def __init__(
         study_id: ObjectId,
         study_file_id: ObjectId,
         *args,
-        **kwargs
+        **kwargs,
```

Comment:
I learned something new -- I see this is recommended by PEP8 for multiline lists, nice